<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>840</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1090</width>
    <height>840</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1090</width>
    <height>840</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TiBoxEditApp</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo_without_text.png</normaloff>logo_without_text.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(226, 226, 226);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QFrame" name="l_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>761</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>681</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>249</width>
        <height>679</height>
       </rect>
      </property>
      <widget class="QTreeWidget" name="day_tw">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>261</width>
         <height>681</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(241, 241, 241)</string>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">Days</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Templates</string>
        </property>
        <item>
         <property name="text">
          <string>Monday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tuesday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wednesday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thursday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Friday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Saturday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sunday</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Added days</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="copyDayButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>710</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pasteDayButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>710</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Paste</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteDayButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>710</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="c_frame">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>471</width>
      <height>761</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>431</width>
       <height>381</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 2px solid rgb(159, 159, 159); 
	border-radius: 3px;
	background: rgb(202, 202, 202);
 } 
QGroupBox::title { 
	background-color: transparent;
	subcontrol-position: top center;
	padding-top: 5px;
 } </string>
     </property>
     <property name="title">
      <string>Subject</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string>Teacher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string>Lesson Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QComboBox" name="subject_type_cb">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QComboBox" name="subject_week_number_cb">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string>Week Type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="subject_time_cb">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="subject_teacher_cb">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="subjectAddButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="subject_description_tb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject_name_le">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QListWidget" name="subject_name_lw">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="subjectEditButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>340</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>431</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(202, 202, 202);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>Start date</string>
      </attribute>
      <widget class="QCalendarWidget" name="start_calendar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background: rgb(255, 255, 255);
border: 0px</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>End date</string>
      </attribute>
      <widget class="QCalendarWidget" name="end_calendar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background: rgb(255, 255, 255);
border: 0px</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>400</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Add End date</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="r_frame">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>351</width>
      <height>761</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>681</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>329</width>
        <height>679</height>
       </rect>
      </property>
      <widget class="QListWidget" name="subject_list">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>681</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="deleteSubjectButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>710</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pasteSubjectButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>710</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Paste</string>
     </property>
    </widget>
    <widget class="QPushButton" name="copySubjectButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>710</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="SaveFileButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>770</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadFileButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>770</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QLabel" name="group_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>800</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of choosen group</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
